### PR TITLE
Update script to use EIP-2470

### PR DESCRIPTION
**Pull request by anhnq82**
https://github.com/derivable-io/create2-vanity-address/issues/1

### CODE REVIEW

File: scripts/UTR.localdeploy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const [owner] = await ethers.getSigners();
+    const signer = owner;
+
+    // Setup to deploy UTR
+    // const url = "https://bsc-dataseed.binance.org/"
+    const singletonFactoryAddress = '0xce0042B868300000d44A59004Da54A005ffdcf9f';
+    const singletonFactoryABI = require('./abi/SingletonFactory.json');
+    const singletonFactory = new ethers.Contract(singletonFactoryAddress, singletonFactoryABI, owner.provider);
+    const initCodeUTR = require('@derivable/utr/build/UniversalTokenRouter.json').bytecode;
+    const salt = '0x46813c3ad3ccf666bcd988ce34c98823d067cd15e337d30d2ac76a189b112553';
```
Comment (Zergity) [inline scripts/UTR.localdeploy.js]: what is this `salt`?

Comment (anhnq82) [reply scripts/UTR.localdeploy.js]: its saltHash
keccak256(salt)

Comment (Zergity) [reply scripts/UTR.localdeploy.js]: then you should write `ethers.utils.keccak256(salt)` for clarification

**Review by Zergity — APPROVED** [review]
please go ahead and take charge of this repo

**Pull request merged**